swf/src/types.rs

```diff
@@ -112,8 +112,19 @@ impl HeaderExt {
         }
     }
 
+    /// Returns the header for the error state movie stub which is used if no file
+    /// could be loaded or if the loaded content is no valid supported content.
+    pub fn default_error_header() -> Self {
```

Comment:
I thought it would be cleaner to have the different default headers that Ruffle uses for an SWF in different states all in the same place. Especially since both the success header (which is used in the success state) and the image header (which is used in the image state) are located here, I think the same makes sense for the error header (used in the error state).

Reply:
Aah, in that case we'll have to keep with what's already there. Maybe a future PR could move that out of `swf`.